editor/editor_plugin_settings.cpp

```diff
@@ -111,19 +101,22 @@ void EditorPluginSettings::update_plugins() {
 				}
 
 				TreeItem *item = plugin_list->create_item(root);
-				item->set_text(0, name);
-				item->set_tooltip_text(0, TTR("Name:") + " " + name + "\n" + TTR("Path:") + " " + path + "\n" + TTR("Main Script:") + " " + scr + "\n" + TTR("Description:") + " " + wrapped_description);
-				item->set_metadata(0, path);
-				item->set_text(1, version);
-				item->set_metadata(1, scr);
-				item->set_text(2, author);
-				item->set_metadata(2, description);
-				item->set_cell_mode(3, TreeItem::CELL_MODE_CHECK);
-				item->set_text(3, TTR("Enable"));
-				bool is_active = EditorNode::get_singleton()->is_addon_plugin_enabled(path);
-				item->set_checked(3, is_active);
-				item->set_editable(3, true);
-				item->add_button(4, get_editor_theme_icon(SNAME("Edit")), BUTTON_PLUGIN_EDIT, false, TTR("Edit Plugin"));
+				item->set_text(COLUMN_NAME, name);
+				if (is_active) {
+					item->set_custom_font(COLUMN_NAME, get_theme_font("bold", EditorStringName(EditorFonts)));
```

Comment:
You could fetch the font before the loop. Same below.

Reply:
The question is whether to do this or make the name more transparent (although in the original proposal no one seemed to mind the bold)

Reply:
You'd still need to fetch a "disabled" color, no?

Reply:
Ever since the massive editor theme refactoring I'm not sure how. Do you have a hint?

Reply:
e.g. `get_theme_color(SNAME("font_disabled_color"), EditorStringName(Editor))`